CHANGELOG.rst

```diff
@@ -1,6 +1,123 @@
+(TODO: where to put this?)
+- Update R batched logging API to accept dataframes (#1214, @dbczumar)
+
+
+Breaking changes:
+
+- API stabilization
+
+    Clean-ups for API stabilization
+    - Stabilizing methods in Python Models modules (#1226, @sueann)
+    - Rename dst_path parameter in pyfunc save_model to path (#1221, @aarondav)
+    - Remove deprecated RunInfo properties from start_run (#1220, @aarondav)
+    - Remove deprecated fields from REST, Python, and R APIs for 1.0 (#1188, @aarondav)
+    (TODO: better description - mostly makes small changes to the APIs)
+    - Mark Python/R model/deploy APIs as stable or experimental for 1.0 (#1222, @sueann)
+
+    - Unify ``mlflow pyfunc`` and ``mlflow rfunc`` CLI (#1257, @tomasatdatabricks)
+    - Change serving docs from "mlflow pyfunc serve" to "mlflow models serve" (#1321, @dbczumar)
+
+    - [Easy] Turn args into opts in experiments CLI commands (#1235, @sueann)
+    - [CLI] clean-ups for 1.0 (#1233, @sueann)
+    - Deprecate --file-store CLI argument to `mlflow server` and `mlflow ui` (#1196, @smurching)
+
+    - R flavors refactor (#1299, @kevinykuo)
+    - Finalize R projects API & update behavior of mlflow_install() (#1265, @smurching)
+    - API cleanups for MLflow R Tracking API (#1246, @smurching)
+    - Rename run_uuid in Python, Java, and REST API to run_id (#1187, @aarondav)
+
+    - Prefix environment variables with "MLFLOW_" (#1268, @aarondav)
+
+(TODO: is this breaking??)
+- Add DB migration support + migration logic for metric x coordinates (#1155, @smurching)
+
+Runs URI (breaking)
+- Cli command to download artifacts from a URI. Updated R apis for loading / scoring models. (#1206, @tomasatdatabricks)
+- [Runs URI][Python] changes in "models" methods (#1190, @dbczumar)
+- [Runs URI][Python] changes in deploy methods (#1174, @sueann)
+- Fix RunsArtifactRepository.list_artifacts (#1175, @sueann)
+- [Runs URI] Add RunsArtifactRepository (#1169, @sueann)
+
+
+Run tags (user in 1.0, name & source in 0.9.0)
+Mention database migration
+(list others in breaking changes; or put this in the breaking changes section but mention in the blog post)
+- Move user attribute to a tag (#1230, @acroz)
+
+(TODO: better descriptions)
+- Don't copy local artifacts in download_artifacts (#1307, @andrewmchen)
+- Update how we handle mlflow dependency when dealing with models.  (#1308, @tomasatdatabricks)
+- Remove handling of dependencies via packrat in R APIs (#1263, @smurching)
+- Simplify "mlflow ui" command and clarify relationship to "mlflow server" (#1267, @aarondav)
+- Run's status converted to string data type (#1264, @mparkhe)
+- Update model flavors to lazily import dependencies (#1238, @dbczumar)
+- Remove default dependencies on boto3, scikit-learn, and mleap (#1223, @aarondav)
+- Remove store argument from get_artifact_repository() (#1138, @sueann)
+- Expose RunData fields (metrics, params, tags) as dictionaries in Python API (#1078, @smurching)
+
+More features and improvements:
```

Comment:
@clemens-db - this PR is not quite ready, but putting up a draft here since the "Breaking Changes" and "More features" sections may be useful for the blog post. I also forgot to mention the Windows support in the earlier version of the doc - we should probably include that as a major feature in the blog post.